blueprints/app/files/package.json

```diff
@@ -26,10 +26,20 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "<% if (!typescript) { %>@babel/eslint-parser": "^7.25.1",
-    "@babel/plugin-proposal-decorators": "^7.24.7",
-    "<% } %>@ember/optional-features": "^2.1.0",
+    "@babel/core": "^7.25.2<% if (!typescript) { %>",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7<% } %><% if (typescript) { %>",
+    "@ember-data/adapter": "~5.4.0-beta.11",
+    "@ember-data/graph": "~5.4.0-beta.11",
+    "@ember-data/json-api": "~5.4.0-beta.11",
+    "@ember-data/legacy-compat": "~5.4.0-beta.11",
+    "@ember-data/model": "~5.4.0-beta.11",
+    "@ember-data/request": "~5.4.0-beta.11",
+    "@ember-data/request-utils": "~5.4.0-beta.11",
+    "@ember-data/serializer": "~5.4.0-beta.11",
+    "@ember-data/store": "~5.4.0-beta.11",
+    "@ember-data/tracking": "~5.4.0-beta.11<% } %>",
```

Comment:
Comment via @runspired: these types are available as of 5.3.8, so we could use that instead of a beta.

Reply:
ember-data below is set to ~5.4.0-beta.11

Reply:
Yes, I just matched the ember-data version that was already present in the blueprints. This is the [PR](https://github.com/ember-cli/ember-cli/commit/891ea487ad30bf53c03b565af355da7819a59cdc#diff-e4c0828d65b51abb30d80befe6fe7870dc36a73dad0083a9dcbb09d48f58e1cfL85) that bumped from 5.3.8 to 5.4.0-beta.11. Should I downgrade everything back to 5.3.8?

Reply:
Sorry, disregard me, as long as this is current version for this branch of ember-cli it's good.